docs/source/developing/writing-docs.adoc

```diff
@@ -77,7 +77,7 @@ http://<IP of Docker daemon>:4567.
 == Deploying Minishift documentation
 
 The Minishift documentation is deployed on _docs.openshift.org_ under
-link:https://docs.openshift.org/minishift[https://docs.openshift.org/minishift].
+link:https://docs.openshift.org/latest/minishift/index.html[https://docs.openshift.org/latest/minishift/index.html].
```

Comment:
That's not quite right, but it will work for now. We cannot assume assume 'latest'. I think best to remove the whole link.